clmm/modeling.py

```diff
@@ -91,7 +91,7 @@ def get_3d_density(r3d, mdelta, cdelta, cosmo, Delta=200, halo_profile_parameter
         Mass overdensity definition; defaults to 200.
     halo_profile_parameterization : str, optional
         Profile model parameterization, with the following supported options:
-        `nfw` (default) - $\rho(r) = \frac{\rho_0}{c/(r/R_{vir})(1+c/(r/R_{vir}))^2}$ [insert citation here]
+        `nfw` (default) - :math:`\rho(r) = \frac{\rho_0}{c/(r/R_{vir})(1+c/(r/R_{vir}))^2}` [insert citation here]
```

Comment:
Let's change this to give options here but descriptions of the options in the main section of the docstring above. This is too much for a parameter description.

Reply:
How about now?